src/common/low_precision_transformations/tests/mark_dequantization_subgraph_transformation.cpp

```diff
@@ -176,6 +176,37 @@ TEST_F(TransformationTestsF, KeepConstPrecision2BranchesSameShapes) {
     comparator.enable(FunctionsComparator::CmpValues::RUNTIME_KEYS);
 }
 
+TEST_F(TransformationTestsF, MarkDequantizationScaleOnTheLeftBranch) {
+    {
+        auto lp_const = std::make_shared<opset10::Constant>(element::u4, Shape{27}, 1);
+        auto scale = opset10::Constant::create(element::i64, Shape{}, {2});
```

Comment:
Minor: i64 precision looks quite unusual from real cases perspective. Can we use e.g. f16 instead? If it is possible of course